### PR TITLE
Inject instance of Admin_AJAX into `Admin_Page` class.

### LINKED ISSUE #118: Inject Admin_AJAX instance into Admin_Page class
### Is your enhancement related to a problem? Please describe.

Inject Admin_AJAX instance into Admin_Page class as part of the constructor. This improves testability and keep it inline with other classes. 

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by spacedmonkey**
### Description of the Change
Inject instance of Admin_AJAX into `Admin_Page` class.

Closes #118

### Changelog Entry

> Added - New feature
> Changed - Existing functionality
> Deprecated - Soon-to-be removed feature
> Removed - Feature
> Fixed - Bug fix
> Security - Vulnerability

### Credits

Props @username, @username2, ...

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my change.
- [x] All new and existing tests pass.

### CODE REVIEW

File: includes/Admin/Admin_Page.php [hunk 1]
```diff
@@ -26,9 +26,11 @@ class Admin_Page {
 	 * Constructor.
 	 *
 	 * @since n.e.x.t
+	 *
+	 * @param Admin_AJAX $admin_ajax    Instance of Admin_AJAX.
```
Comment (felixarntz) [inline includes/Admin/Admin_Page.php]:
```suggestion
	 * @param Admin_AJAX $admin_ajax Instance of Admin_AJAX.
```

File: includes/Plugin_Main.php [hunk 2]
```diff
@@ -59,8 +60,9 @@ public function add_hooks() {
 			require_once WP_PLUGIN_CHECK_PLUGIN_DIR_PATH . 'cli.php';
 		}
 
+		$admin_ajax = new Admin_AJAX();
```
Comment (felixarntz) [inline includes/Plugin_Main.php:63]: Nit-pick: This doesn't need to be in a variable, we could simply use `new Admin_Page( new Admin_AJAX() )`.

**Review by felixarntz — APPROVED** [review]
@spacedmonkey LGTM, just two nit-picks here.

File: includes/Plugin_Main.php (hunk 2 above)
Comment (spacedmonkey) [reply includes/Plugin_Main.php:63]: I find this more readable personally. I can change it you really hate it.

**Review by mukeshpanchal27 — APPROVED** [review]

**Review by jjgrainger — APPROVED** [review]

**Pull request merged**